src/compressor.js

```diff
@@ -0,0 +1,10 @@
+import JXG from "./jxg";
```

Comment:
These top-level files sub in for the old build files. Users can also do

```
import JXG from "jsxgraph/src/compressor"
```

if they are using the npm package (I think that's the right syntax?)